runtime/doc/options.txt

```diff
@@ -5571,6 +5571,15 @@ A jump table for the options with a short description can be found at |Q_op|.
 	NOTE: This option is set to the Vi default value when 'compatible' is
 	set and to the Vim default value when 'compatible' is reset.
 
+                                                *'relativenumberformat'* *'rnfm'*
```

Comment:
This block should be moved after `*'relativenumber'*`. (Should be in alphabetical order.)